specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -3181,6 +4633,41 @@
         }
       }
     },
+    "WorkspaceConnectionOAuth2": {
+      "description": "ClientId and ClientSecret are required. Other properties are optional\r\ndepending on each OAuth2 provider's implementation.",
+      "type": "object",
+      "properties": {
+        "authUrl": {
+          "description": "Required by Concur connection category",
+          "type": "string"
+        },
+        "clientId": {
+          "type": "string"
+        },
+        "clientSecret": {
+          "type": "string"
+        },
+        "developerToken": {
```

Comment:
>developerToken [](http://example.com/codeflow?start=9&length=14)

If this is a secret, it's not allowed in GET payloads and must be tagged with x-ms-secret. This type is currently present in GET payloads.